api/src/middleware/locale.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  getLocale: (req, res, next) => {
+    req.locale = req && req.headers && req.headers['accept-language'];
```

Comment:
If the user has explicitly chosen a locale it'll be stored in the cookie. I think it'd be better to use that value first and only fall back to the "accept-language" if needed.

Reply:
Great suggestion, done!